NeuralAmpModeler/resources/main.rc

```diff
@@ -1,250 +1,289 @@
+    COMBOBOX        IDC_COMBO_AUDIO_IN_R,65,126,40,200,CBS_DROPDOWNLIST | CBS_HASSTRINGS
+    LTEXT           "Input 2 (R)",IDC_STATIC,65,115,34,8
+    COMBOBOX        IDC_COMBO_AUDIO_OUT_L,20,155,40,200,CBS_DROPDOWNLIST | CBS_HASSTRINGS
+    LTEXT           "Output 1 (L)",IDC_STATIC,20,145,38,8
+    COMBOBOX        IDC_COMBO_AUDIO_OUT_R,65,155,40,200,CBS_DROPDOWNLIST | CBS_HASSTRINGS
+    LTEXT           "Output 2 (R)",IDC_STATIC,65,145,40,8
+    GROUPBOX        "MIDI Device Settings",IDC_STATIC,5,190,210,85
+    COMBOBOX        IDC_COMBO_MIDI_OUT_DEV,15,250,100,200,CBS_DROPDOWNLIST | CBS_HASSTRINGS
+    LTEXT           "Output Device",IDC_STATIC,15,240,47,8
+    COMBOBOX        IDC_COMBO_MIDI_IN_DEV,15,220,100,200,CBS_DROPDOWNLIST | CBS_HASSTRINGS
+    LTEXT           "Input Device",IDC_STATIC,15,210,42,8
+    LTEXT           "Input Channel",IDC_STATIC,125,210,45,8
+    COMBOBOX        IDC_COMBO_MIDI_IN_CHAN,125,220,50,200,CBS_DROPDOWNLIST | CBS_HASSTRINGS
+    LTEXT           "Output Channel",IDC_STATIC,125,240,50,8
+    COMBOBOX        IDC_COMBO_MIDI_OUT_CHAN,125,250,50,200,CBS_DROPDOWNLIST | CBS_HASSTRINGS
+END
+
+IDD_DIALOG_MAIN DIALOG 0, 0, 500, 300
+STYLE DS_SETFONT | DS_MODALFRAME | DS_CENTER | WS_MINIMIZEBOX | WS_POPUP | WS_CAPTION | WS_SYSMENU
+CAPTION "NeuralAmpModeler"
+MENU IDR_MENU1
+FONT 8, "MS Sans Serif"
+BEGIN
+END
+
+
+/////////////////////////////////////////////////////////////////////////////
+//
+// Menu
+//
+
+IDR_MENU1 MENU
+BEGIN
+    POPUP "&File"
+    BEGIN
+        MENUITEM "&Preferences...\tCtrl+,",     ID_PREFERENCES
+        MENUITEM "&Quit",                       ID_QUIT
+    END
+    POPUP "&Debug"
+    BEGIN
+        MENUITEM "&Live Edit Mode\tCtrl+E",     ID_LIVE_EDIT
+        MENUITEM "&Show Control Bounds\tCtrl+B", ID_SHOW_BOUNDS
+        MENUITEM "&Show Drawn Area\tCtrl+D",    ID_SHOW_DRAWN
+        MENUITEM "&Show FPS\tCtrl+F",           ID_SHOW_FPS
+    END
+    POPUP "&Help"
+    BEGIN
+        MENUITEM "&About",                      ID_ABOUT
+        MENUITEM "&Read Manual",                ID_HELP
+    END
+END
+
+
+/////////////////////////////////////////////////////////////////////////////
+//
+// DESIGNINFO
+//
+
+#ifdef APSTUDIO_INVOKED
+GUIDELINES DESIGNINFO
+BEGIN
+    IDD_DIALOG_PREF, DIALOG
+    BEGIN
+    END
+
+    IDD_DIALOG_MAIN, DIALOG
+    BEGIN
+    END
+END
+#endif    // APSTUDIO_INVOKED
+
+
+/////////////////////////////////////////////////////////////////////////////
+//
+// AFX_DIALOG_LAYOUT
+//
+
+IDD_DIALOG1 AFX_DIALOG_LAYOUT
+BEGIN
+    0
+END
+
+
+#ifdef APSTUDIO_INVOKED
+/////////////////////////////////////////////////////////////////////////////
+//
+// TEXTINCLUDE
+//
+
+1 TEXTINCLUDE 
+BEGIN
+    "resource.h\0"
+END
+
+2 TEXTINCLUDE 
+BEGIN
+    "#include ""winres.h""\r\n"
+    "\0"
+END
+
+3 TEXTINCLUDE 
+BEGIN
+    "#include ""..\\config.h""\r\n"
+    "ROBOTO_FN TTF ROBOTO_FN\r\n"
+    "RONDUIT_FN TTF RONDUIT_FN\r\n"

```

Comment:
Tested on Windows; just this (to `MICHROMA_FN` and it's good. Let me see if I can grab it.

Reply:
Got it